CircleCi nightly feature-update-regression test fails at run-update-feature step

**Summary of the defect**



**How to reproduce (if possible)**

https://app.circleci.com/pipelines/github/hashgraph/hedera-services/6212/workflows/25dbc5e7-7abf-45fe-a15c-5c21cfa2933a/jobs/40288

**Service logs (if applicable)**
```
...

2020-10-30 08:27:22.339 INFO   227  HapiApiSpec - 'CreatePayerAccountForEachClient' finished initial execution of CustomSpecAssert{}
2020-10-30 08:27:22.341 INFO   244  HapiApiSpec - 'CreatePayerAccountForEachClient' - final status: PASSED!
2020-10-30 08:27:22.341 INFO   168  CryptoCreateForSuiteRunner - -------------- RESULTS OF CryptoCreateForSuiteRunner SUITE --------------
2020-10-30 08:27:22.342 INFO   170  CryptoCreateForSuiteRunner - Spec{name=CreatePayerAccountForEachClient, status=PASSED}
2020-10-30 08:27:24.349 INFO   137  FreezeSuite - -------------- STARTING Freeze SUITE --------------
2020-10-30 08:27:24.354 INFO   91   FreezeSuite - Creating zip file from updateSettings
2020-10-30 08:27:24.356 INFO   70   ZipUtil - Root = src/main/resource/testfiles/updateFeature/updateSettings, current = src/main/resource/testfiles/updateFeature/updateSettings
2020-10-30 08:27:24.356 INFO   86   ZipUtil - Current directory src/main/resource/testfiles/updateFeature/updateSettings
2020-10-30 08:27:24.356 INFO   100  ZipUtil - Adding file:/delete.txt
2020-10-30 08:27:24.358 INFO   70   ZipUtil - Root = src/main/resource/testfiles/updateFeature/updateSettings, current = src/main/resource/testfiles/updateFeature/updateSettings/sdk
2020-10-30 08:27:24.358 INFO   79   ZipUtil - Adding dir sdk/
2020-10-30 08:27:24.358 INFO   86   ZipUtil - Current directory src/main/resource/testfiles/updateFeature/updateSettings/sdk
2020-10-30 08:27:24.359 INFO   100  ZipUtil - Adding file:/sdk/new2.txt
2020-10-30 08:27:24.359 INFO   100  ZipUtil - Adding file:/sdk/settings.txt
2020-10-30 08:27:24.359 INFO   100  ZipUtil - Adding file:/sdk/new1.txt
2020-10-30 08:27:24.360 INFO   100  ZipUtil - Adding file:/exec.sh
2020-10-30 08:27:24.361 INFO   96   FreezeSuite - Uploading file Archive.zip
2020-10-30 08:27:24.366 INFO   706  TestHelper - In Else fileName: src/main/resource/application.properties
2020-10-30 08:27:24.367 INFO   720  TestHelper - properties: {UPDATE_DURATION_VALUE=7885000, host4=34.227.84.4, host1=18.232.92.171, ACCOUNT_DURATION=7890000, host3=54.159.114.82, host2=18.234.154.244, retrieveReceipt=true, COUNT_SIZE=1000, NODE_ACCOUNT_NUMBER=3, FILE_DURATION=7890000, CONTRACT_DURATION=7890000, BATCH_SIZE=1000, LOCAL_CALL_GAS=260000, port=50211, TPS=100, defaultListeningNodeAccount=0.0.3, host=18.232.92.171, NODE_REALM_NUMBER=0, startUpFile=src/main/resource/StartUpAccount.txt, NODE_SHARD_NUMBER=0}
2020-10-30 08:27:24.367 INFO   706  TestHelper - In Else fileName: src/main/resource/application.properties
2020-10-30 08:27:24.368 INFO   373  HapiApiSpec - CI_PROPERTIES_MAP: 
2020-10-30 08:27:24.369 INFO   48   MapPropertySource - Initializing a MapPropertySource from {nodes=18.232.92.171:0.0.3:50211,18.234.154.244:0.0.4:50211,54.159.114.82:0.0.5:50211,34.227.84.4:0.0.6:50211, default.node=0.0.3, tls=OFF, txn=ALTERNATE, node.selector=FIXED, default.payer=0.0.1099, ci.properties.map=}
2020-10-30 08:27:24.384 INFO   137  HapiApiClients - Constructed 56 new stubs building clients for HapiApiClients{node0=NodeConnectInfo{host=18.232.92.171, port=50211, tlsPort=50212, account=0.0.3}, node1=NodeConnectInfo{host=18.234.154.244, port=50211, tlsPort=50212, account=0.0.4}, node2=NodeConnectInfo{host=54.159.114.82, port=50211, tlsPort=50212, account=0.0.5}, node3=NodeConnectInfo{host=34.227.84.4, port=50211, tlsPort=50212, account=0.0.6}, default=0.0.3}
2020-10-30 08:27:24.433 INFO   130  FeesAndRatesProvider - The exchange rates are :: [Current hbar/cent equiv 1 <-> 12 expiry @ 4102444800, Next hbar/cent equiv 1 <-> 15 expiry @ 4102444800]
2020-10-30 08:27:24.460 INFO   148  FeesAndRatesProvider - The fee schedule covers 51 ops.
2020-10-30 08:27:24.515 INFO   140  HapiTxnOp - 'uploadFileAndUpdate' -  submitting HapiFileUpdate{node=0.0.3, fileName=UPDATE_FEATURE, path=Archive.zip} via body=transactionID { transactionValidStart { seconds: 1604046384 nanos: 212 } accountID { accountNum: 1099 } } nodeAccountID { accountNum: 3 } transactionFee: 1189689212 transactionValidDuration { seconds: 120 } memo: "\303\203\302\256\303\202\302\267\303\203\302\271tF8\303\202\302\256J\303\203\302\213\303\203\302\220\303\203\302\216" fileUpdate { fileID { fileNum: 150 } contents: "PK\003\004\024\000\b\b\b\000lC^Q\000\000\000\000\000\000\000\000\000\000\000\000\v\000\000\000/delete.txt\313K-7\324+\251(\341\322\323\323/N\311\326\317\313\317K\255\310,.\001\213\345\245\226\033\201\030\000PK\a\b\346\203O\222!\000\000\000%\000\000\000PK\003\004\024\000\b\b\b\000lC^Q\000\000\000\000\000\000\000\000\000\000\000\000\004\000\000\000sdk/\003\000PK\a\b\000\000\000\000\002\000\000\000\000\000\000\000PK\003\004\024\000\b\b\b\000lC^Q\000\000\000\000\000\000\000\000\000\000\000\000\r\000\000\000/sdk/new2.txtKL\204\000\256$(\340J\206\002\256\024(\340\002\000PK\a\bc0jZ\021\000\000\000$\000\000\000PK\003\004\024\000\b\b\b\000lC^Q\000\000\000\000\000\000\000\000\000\000\000\000\021\000\000\000/sdk/settings.txtu\221\315N\303@\f\204\357y\212<@\024\205\242\2263RAph)\244p\267\022\'\261\232\356\256l\'\264}z\274)\342\237\253g<;\376\266\352\260\332\225\324:\254K\005\305[\366\373%\311.K\213\244\342cP\237WaXR\213\242\333\216\021\352\'P\362&\347\027_\f/\310\324\020\3627\213\031d\274\016\001]=\305\311xK=\256a\217Y\272\002rk\324\370\246$\265\177\016.DO\303\210\',Q\225\\+9TJ#\306\244\037\202E\336\371\201\263t\361\207\262\"g+\277\302D\201\365\237\255I\233\366\212d\017\207\207A[oByt\225d\351,a?\270Zn\016\201\030\355\230yQ$\322\371\327{\247\310\016\372\351\214XS\"\303\374\354\266ao\335g\363\367\251\300\210\023\343\215\301\362\226r\031S\316\322\347\a\234\341[{\355\330\253\366x\205\ae\210\300g\037\263-\203\223\b\307\273\307\001\a,\351d/-\n\vl`\207\033`\203l\325\262tIM\023\211K\222\274\001PK\a\b2\345q\234\031\001\000\000\353\001\000\000PK\003\004\024\000\b\b\b\000lC^Q\000\000\000\000\000\000\000\000\000\000\000\000\r\000\000\000/sdk/new1.txt34\204\001.#8\3402\206\003.\0238\340\002\000PK\a\b\232\337\356y\021\000\000\000,\000\000\000PK\003\004\024\000\b\b\b\000lC^Q\000\000\000\000\000\000\000\000\000\000\000\000\b\000\000\000/exec.sh\255Xms\332H\022\376\256_\321\'\303\222d\227W\347\266n\275\245\\y\2018\334\032p\031\274\311\276\025\036K\003L\2204\272\031\3118\233\315\375\366\353\036IH\030\260\235\335P\345B\226z\272\237\356~\372E\034\375\243\231h\325\274\021a\223\207\267p\303\364\322\262\216\254#\230\260 \3629hW\211(\0069\207\245\\C,!\211<\026sx\317\024\314\205\3175$Z\204\213\374\366\234\2638Q\334h8U\213$\340a\214\227\000\2256\204\322\343 <\b\223\340\206+\024\2614\217\241\316\373\226\345*<\307g!_\317<\241\270\033K\365\001\236=\207\217\026\036\205#x\313D\fs\251\000\2150\304\310\341\206\223U\215\202\334\003/Q\364\337\\q\376\aB\216\331\202C\274\344!\250$\244\213\334\v\204\357\312\350\003ll\030\365\332\347<\202o[Vf,\005\003\f\020\016\254\245Z\211EX\034\001\221\252D4\0013\201y\303\"q\032E\235F\253\336iuZ\255\357\216[\323\366\313\343V\253e\024n95\vY\300\035\273t\304\2766\201\373\272\372s5\250z\323\352\233\352\260:\271N\201-\271\357\317\270\273\224P9\037\214\372\2431TZ`w\t\037\371\273\301\207\327\005\276\312\256A\333\250\vVx\027\352\0214\032\315=B\326C6\177H\204\357\231\210\177\bn\244/\\\360E\270\242\200\3565\227E\362\206Nm\0371O\\\0171\230+\025@\275\034As\323\017\241\256\367)\276/\210z\352\017\302\356b\266\t5\277\023\332\204\214(K\250?3tn\004u\005\215\346\203\2413^\035~|\224\031\002\231\304Qb\310\230]\371rA<\"V\021\256mv\2129\374J1\252L\372\227?\r\272\375\331\371\370\354\345\177:\360\373\367\206\342F\206>\343\253\351\305\325\324\331\217\240\271\\\270\2145\320\220\221\347\276\346O<\230B\274w~.,\353\223e\2455?\323\336j\226\266\202R\305\272K\356\256\b<y\204\022i\344\213P\233\214\224\034\364 \346A\324$\311\373\236\035\310\355k\231\204\336\226z\r\366c\207rB\244\342s%\203\302.u\207D)\354X\273\324(\024\247T\310\0175_ +vcz\300\372H\336\303;7>\330yP)\246\024u\035+\347\332\\][\326h\334\353\317\006=\247\322\266\254m\01686&\344\345\373N]su+\\\256\353\212/\024\327Z\310\260q\027\370\266u\361\266\347\\Gk\357\332z\n\2152&T\360T)\341\033\307\312\217whA\224\240\246\017g<\344\212:\032\261\032]\321\246\031Kt\325\367q\202\244\210\263\356\231\236\300\310`g\340\233\271`\206E\326\261M\371\320\235N\256\v\317\024\321\335\020\216\334\373\025\354J\036>\033\034\aj\347\"L\356j\360\373}F\245\211y\2266^\273\372s\275\032\324\253\036T\247\215\352\361\310~\016\203\321\353\261AAv\355:\214\372\323\267\343\313\037gW\027\275\323i\037F4\310*Y^l\264zl\303\253WPI\003\224\261a?\242\036Sk\021~\026$\234#_\bR\211\240%s_Bw\312\335#\b\330\n\207-.\000yk3UOL0\365\016\022}\306\000h\312\251\313\302P\3068\220C\354\314\t6\3114b\304\321T\363v\220\016\224T.k\364\353\206]\320\325\210\332\375\313\313\361\345I\016\'\227\366$\026\037Y7\307\032\306\037\256\224TE\213\274\023\261!\365\025\026\214s\275^J\026\210k\353\000\212I\314T\214\313\223\273ZHE%M{T\306a\373\320\241\274\331$X\276 p\340\221)\233\3028\027\250\342b\320\263\"%\261\254\365\300s*\317\"\215\273\322\034\376\004,\361\bj\256\f\032z-\224\357\351F\344\263\230*\252\361\203\222k\324V\313\245\352\267P\243\v\272\301\326+\250}\204\bW\245xN\231\376T{~\b\332\233\263.\216Y\310\315c\363\331 1\000\263\025i\357\024\335\273\314\355\314\n\213\324x<\246\201\230\a\"w\003\030\272\217}\314\004\365=\273e9\020\353\351e~Da\235\'>\320\216\351\n\345\"\017]\001\247o\'\200{\256P2\244\325\024n\231\022\354\306\347\245&\202\004\374_\263A\t\234\241YC\340\235j\3252Q.\337\221\333S\026%\275!\002\377\330\035\314\020\303\'{\217\316\375\251\270L\302\220\342+C\350\016.\273\347\375\356 \333\257\350\363zp\336wl\332\212\233,\212t\363\r\367\260\373R\3056pC/\246\326e\177t:\334\2264)\336\226:\312\av:\235\2504\031\256\273\333J\277\301\324\230eL\230\035\246\320\263Q\263\2315\204ng\234?\340\353\224\366j\nd\241\024S\317\353\246J\277Ad\334\313\215\323\346I\352\033\366\226b$O%\365u\353vp\233\201\271\377\220*\274\360\336e\276\017=~;\306Z\313\006\020\366+\016\377~4I\031Y\t\367\305\005d\363\270\264\215\230W\214\366?\321\306\232^e6L\327\354\226\027/5\310\243\210+\377Cq.\361d\256\rh\330bQ\346\205Qp\r:\257\276j[O\336BJ\2142\225\363\367a\256\004\306\255h\021\326c\b6\257sY\237\301d\3767\021\361.\220G5\225\003\037Ev9\231*sS\243f\344\v\n\302Z\304\313\302\251bY\2029\256&\273\364}d\343\246\017=z\337y7<w\356IoI\231\036V\177\367\356\344\353\253\320\227\356\252\177\207\361\023\324\200\230\377\323p\034\305\210Ag\002\232\377r\3265\327]\031\272g\335\351\022\333\251\247\235\366K\370mKi.~\316\324\202_\340fD\032\202\273\357\376\265\240o\335n-\214\310/C\246V8\237\334\325$b.?\027\201\210\235\366\267\351\303!\273\353\231\026=\344\001v\351\211\370\203;\307\235\305\036KC\0363M\n\214L\273\325\032\322}\234\226\'\v\367\005\376\321\340\304;=\263\3375\260s\314\305\"\301=\020={\215\025\355<\274\251\3367\210Kv\315\264*_\3344_\324\340\241a\207\205\220Nw\003\201J\001\276\302i\244\345\032_#\227\305\306S\256\214\")\275\271\317\026N\333\230\314-`\3339)\254?h\274\334H\262\257\317Z\375\036/\321\200\271\343\211]\372\241\342\351\345i\035\254\315\277R\227\3335\371\304z\374r\201\336\344\357\200eHB\252q\234\020\343\t\234T\362\363\366\3666g\375\037PK\a\byt\342\212\276\006\000\000\360\022\000\000PK\001\002\024\000\024\000\b\b\b\000lC^Q\346\203O\222!\000\000\000%\000\000\000\v\000\000\000\000\000\000\000\000\000\000\000\000\000\000\000\000\000/delete.txtPK\001\002\024\000\024\000\b\b\b\000lC^Q\000\000\000\000\002\000\000\000\000\000\000\000\004\000\000\000\000\000\000\000\000\000\000\000\000\000Z\000\000\000sdk/PK\001\002\024\000\024\000\b\b\b\000lC^Qc0jZ\021\000\000\000$\000\000\000\r\000\000\000\000\000\000\000\000\000\000\000\000\000\216\000\000\000/sdk/new2.txtPK\001\002\024\000\024\000\b\b\b\000lC^Q2\345q\234\031\001\000\000\353\001\000\000\021\000\000\000\000\000\000\000\000\000\000\000\000\000\332\000\000\000/sdk/settings.txtPK\001\002\024\000\024\000\b\b\b\000lC^Q\232\337\356y\021\000\000\000,\000\000\000\r\000\000\000\000\000\000\000\000\000\000\000\000\0002\002\000\000/sdk/new1.txtPK\001\002\024\000\024\000\b\b\b\000lC^Qyt\342\212\276\006\000\000\360\022\000\000\b\000\000\000\000\000\000\000\000\000\000\000\000\000~\002\000\000/exec.shPK\005\006\000\000\000\000\006\000\006\000V\001\000\000r\t\000\000\000\000" }; sigs=sigPair { pubKeyPrefix: "\n" ed25519: "L\2603{\214\225\264d*V\304\"\334v\204V|\343Q\216\0249\000\351l\242\347xp\277J6\334r\265\360\312\366V\247\350\311\177\365y&l\245_h\273]\a\217\216#\311\n\255\313`\v\337\f" }
2020-10-30 08:27:24.523 WARN   199  HapiSpecOperation - 'uploadFileAndUpdate' - HapiFileUpdate{sigs=1, node=0.0.3, fileName=UPDATE_FEATURE, path=Archive.zip} failed!
java.lang.AssertionError: Wrong precheck status! expected:<OK> but was:<AUTHORIZATION_FAILED>
	at org.junit.Assert.fail(Assert.java:88) ~[junit-4.12.jar:4.12]
	at org.junit.Assert.failNotEquals(Assert.java:834) ~[junit-4.12.jar:4.12]
	at org.junit.Assert.assertEquals(Assert.java:118) ~[junit-4.12.jar:4.12]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:199) ~[classes/:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[classes/:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.exec(HapiApiSpec.java:222) ~[classes/:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.run(HapiApiSpec.java:176) ~[classes/:?]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1654) ~[?:?]
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658) ~[?:?]
	at com.hedera.services.bdd.suites.HapiApiSuite.runSync(HapiApiSuite.java:249) ~[classes/:?]
	at com.hedera.services.bdd.suites.HapiApiSuite.runSuite(HapiApiSuite.java:140) ~[classes/:?]
	at com.hedera.services.bdd.suites.HapiApiSuite.runSuiteSync(HapiApiSuite.java:133) ~[classes/:?]
	at com.hedera.services.bdd.suites.SuiteRunner.lambda$runSuitesSync$15(SuiteRunner.java:463) ~[classes/:?]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:176) ~[?:?]
	at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948) ~[?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[?:?]
	at com.hedera.services.bdd.suites.SuiteRunner.runSuitesSync(SuiteRunner.java:464) ~[classes/:?]
	at com.hedera.services.bdd.suites.SuiteRunner.lambda$runTargetCategories$10(SuiteRunner.java:438) ~[classes/:?]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195) [?:?]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1654) [?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) [?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) [?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) [?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) [?:?]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) [?:?]
	at com.hedera.services.bdd.suites.SuiteRunner.runTargetCategories(SuiteRunner.java:438) [classes/:?]
	at com.hedera.services.bdd.suites.SuiteRunner.runCategories(SuiteRunner.java:407) [classes/:?]
	at com.hedera.services.bdd.suites.SuiteRunner.main(SuiteRunner.java:331) [classes/:?]
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:254) [exec-maven-plugin-3.0.0.jar:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-10-30 08:27:24.528 INFO   244  HapiApiSpec - 'uploadFileAndUpdate' - final status: FAILED!
2020-10-30 08:27:24.529 INFO   168  FreezeSuite - -------------- RESULTS OF Freeze SUITE --------------
2020-10-30 08:27:24.529 INFO   170  FreezeSuite - Spec{name=uploadFileAndUpdate, status=FAILED}
2020-10-30 08:27:24.533 INFO   412  SuiteRunner - ============== sync run results ==============
2020-10-30 08:27:24.533 INFO   415  SuiteRunner - FreezeSuite                      :: 0/1 suites ran OK
2020-10-30 08:27:24.534 INFO   421  SuiteRunner -   --> Problems in suite 'Freeze' :: Spec{name=uploadFileAndUpdate, status=FAILED}
Ubuntu 18.04.2 LTS





```
**Environment:**
- OS: [e.g. Ubuntu 18.04]
- Java: [e.g. OpenJDK 11.0.4]
- Hedera Services Version: [e.g. 0.0.5]
- HAPI Version: [e.g. 0.0.5]

**Additional Context**
Add any other context about the problem here. Attach any logs here, if applicable.

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -2139,7 +2139,6 @@ jobs:
             '/repo/.circleci/scripts/show_java_process.sh'
       - run-eet-suites:
           dsl-args: FreezeSuite
-          ci-properties-map: default.payer=0.0.2
       - run:
           name: Show logs before HGCApp restart
           command: |

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -911,7 +911,7 @@ workflows:
           filters:
             branches:
               only:
-                - master
+                - Disable_4N_Perf_JRS_Regression_master
     jobs:
       - build-platform-and-services
       - run-performance-regression:
@@ -1154,20 +1154,16 @@ workflows:
             - attach_workspace:
                 at: /
   feature-update-regression:
-#    triggers:
-#        - schedule:
-#            cron: "0 8 * * *"
-#            filters:
-#              branches:
-#                only:
-#                  - master
+    triggers:
+        - schedule:
+            cron: "0 8 * * *"
+            filters:
+              branches:
+                only:
+                  - master
     jobs:
       - fast-build-artifact:
           context: Slack
-          filters:
-            branches:
-              only:
-                - 00715-D-nightly-update-regression-failure
           post-steps:
             - save-this-job-client-logs:
                 workflow-name: "feature-update-regression"
@@ -1326,7 +1322,6 @@ workflows:
               ignore:
                 - /.*-PERF/
                 - /.*-REGRESSION/
-                - 00715-D-nightly-update-regression-failure
           workflow-name: "Continuous integration"
       - start-singlejob-testnet:
           context: Slack

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -1154,16 +1154,20 @@ workflows:
             - attach_workspace:
                 at: /
   feature-update-regression:
-    triggers:
-        - schedule:
-            cron: "0 8 * * *"
-            filters:
-              branches:
-                only:
-                  - master
+#    triggers:
+#        - schedule:
+#            cron: "0 8 * * *"
+#            filters:
+#              branches:
+#                only:
+#                  - master
     jobs:
       - fast-build-artifact:
           context: Slack
+          filters:
+            branches:
+              only:
+                - 00715-D-nightly-update-regression-failure
           post-steps:
             - save-this-job-client-logs:
                 workflow-name: "feature-update-regression"
@@ -1322,6 +1326,7 @@ workflows:
               ignore:
                 - /.*-PERF/
                 - /.*-REGRESSION/
+                - 00715-D-nightly-update-regression-failure
           workflow-name: "Continuous integration"
       - start-singlejob-testnet:
           context: Slack
@@ -2134,6 +2139,7 @@ jobs:
             '/repo/.circleci/scripts/show_java_process.sh'
       - run-eet-suites:
           dsl-args: FreezeSuite
+          ci-properties-map: default.payer=0.0.2
       - run:
           name: Show logs before HGCApp restart
           command: |

--- a/test-clients/src/main/java/com/hedera/services/bdd/suites/freeze/FreezeSuite.java
+++ b/test-clients/src/main/java/com/hedera/services/bdd/suites/freeze/FreezeSuite.java
@@ -106,9 +106,11 @@ public class FreezeSuite extends HapiApiSuite {
 		return defaultHapiSpec("uploadFileAndUpdate")
 				.given(
 						fileUpdate(fileIDString).path(uploadFile)
+						.payingWith(GENESIS)
 				).when(
 						freeze().setFileID(fileIDString)
 								.setFileHash(hash)
+								.payingWith(GENESIS)
 								.startingIn(60).seconds().andLasting(1).minutes()
 				).then(
 				);

